cloup/formatting/_util.py

```diff
@@ -23,5 +24,9 @@ def ensure_is_cloup_formatter(formatter: click.HelpFormatter) -> 'cloup.HelpForm
     raise TypeError(FORMATTER_TYPE_ERROR)
 
 
-def unstyled_len(string: str) -> int:
-    return len(click.unstyle(string))
+def width(string: Any) -> int:
```

Comment:
There's a high change this function name will collide with a local variable. Also why the `Any`? Can you use `wcswidth` directly?

Reply:
`Any` just because that's what the `wcswidth` takes (and `len`, more or less). It will convert it to a string if necessary. Might as well be general if we can, right? :-)

Reply:
I don't follow you. `wcswidth` [wants a string](https://github.com/jquast/wcwidth/blob/4d4548048275ee25ba698cb0bd411296ff3f2238/wcwidth/wcwidth.py#L226), the argument name is `string` and the function makes sense only for strings AFAICS. So I'd change the parameter type to `str` and remove the conversion to string.

Reply:
@janluke Oh, there aren't any type annotations. VS Code was looking like it was telling me that there was, and that it was `Any`. That's fair enough then.